app/src/main/java/com/example/coronadiagnosticapp/data/network/NetworkDataSourceImpl.kt

```diff
@@ -70,7 +70,7 @@ class NetworkDataSourceImpl @Inject constructor(val api: ApiServer) : NetworkDat
     }
 
     override suspend fun getQuestions(): List<JsonObject> {
-        val language = Locale.getDefault().language
+        val language = Locale.getDefault().language.replace("iw", "he")
```

Comment:
why replacing? isn't iw, and he are same ? I know that iw is the 'older' code for Hebrew

Reply:
The server knows "he" not "iw"